### PR TITLE
Added Property Identifier in dashboard activity stream

### PR DESCRIPTION
**Pull request by kotwani2883**
## Explain the feature/fix

Added Property Identifier in dashboard activity stream before the application identifier so that one can identify to which property the application belong to . Also added the links to navigate there, 

## Does this PR introduce a breaking change

No

[screen-capture (12).webm](https://user-images.githubusercontent.com/56580582/212934035-f79b71e6-fd95-43e9-9158-06d417aef50b.webm)

### CODE REVIEW

File: packages/manager/src/views/DashboardPage/components/ActivityStreamDashboard/ActivityStreamDashboard.tsx [hunk 1]
```diff
@@ -18,21 +21,21 @@ export const ActivityStreamDashboard = () => (
...
       }}
       isRounded
       className={css('pf-u-px-sm rounded-md transition hover:shadow-sm')}
     >
-      <SimpleBarReact style={{ maxHeight: 650, padding: '10px' }}>
+      <SimpleBarReact style={{ maxHeight: 830, padding: '10px' }}>
         <div style={{ marginTop: '20px', marginLeft: '10px' }}>
-          <ActivityStream propertyIdentifier="" action="APPLICATION_DEPLOYED" />
+          <ActivityStream
+            propertyIdentifier=""
```
Comment (akhilmhdh) [inline packages/manager/src/views/DashboardPage/components/ActivityStreamDashboard/ActivityStreamDashboard.tsx:33]: edit the activity stream component and change the these two props to not required with default value as empty string. You can then avoid using this props here

**Review by akhilmhdh — CHANGES_REQUESTED** [review]
Bump it up, bump it up, bump the package.json. 

Bump it up, bump it up, bump the package-lock.json.

**Review by akhilmhdh — APPROVED** [review]

**Review by hybridx — APPROVED** [review]

**Review by soumyadip007 — APPROVED** [review]
LGTM

**Comment by soumyadip007** [comment]
@kotwani2883 could you please resolve this conflict

**Pull request merged**